### ISSUE TITLE
schema.org/docs/jsonldcontext.jsonld is now being returned as application/octet-stream instead of application/ld+json

### ISSUE THREAD

**Issue by Sarke** [schemaorg/schemaorg#2595]
```
$ curl -I https://schema.org/docs/jsonldcontext.jsonld
HTTP/2 200 
access-control-allow-credentials: true
access-control-allow-headers: Accept
access-control-allow-origin: \*
access-control-allow-methods: GET
date: Tue, 02 Jun 2020 10:32:02 GMT
expires: Tue, 02 Jun 2020 10:42:02 GMT
etag: "nFpXbg"
x-cloud-trace-context: ad08e0ade3623eba5d1d483500795e0d
content-type: application/octet-stream
server: Google Frontend
content-length: 155833
age: 18
cache-control: public, max-age=600
alt-svc: h3-27=":443"; ma=2592000,h3-25=":443"; ma=2592000,h3-T050=":443"; ma=2592000,h3-Q050=":443"; ma=2592000,h3-Q049=":443"; ma=2592000,h3-Q048=":443"; ma=2592000,h3-Q046=":443"; ma=2592000,h3-Q043=":443"; ma=2592000,quic=":443"; ma=2592000; v="46,43"
```

Related to #2578?

**Comment by davidlehn** [comment]
In addition to content-type, looks like the build system is doing something odd.  Sources have `access-control-allow-origin` as `"*"`, but all the static files have the invalid `\*`.
https://github.com/schemaorg/schemaorg/search?q=access-control-allow-origin&unscoped_q=access-control-allow-origin
This is causing browser problems with https://json-ld.org/playground/:
```
Access to XMLHttpRequest at 'https://schema.org/' from origin 'https://json-ld.org' has been blocked by CORS policy: The 'Access-Control-Allow-Origin' header contains the invalid value '\*'.
```

**Comment by RichardWallis** [comment]
"*" appearing as \* - Fixed in commit e2683d58c9d9f3625751b8bfc8b3eec9af569c3b
Mime_type - Fixed in commit a9a3ff767b33ba97a106daae9c8e2ca6010398c4

**Comment by RichardWallis** [comment]
Fixed in deployment to schema.org 2020-06-03T12:00:00Z
